smoke-test.sh

```diff
@@ -21,6 +21,8 @@ disk_usage () { du -t 10K -h "$HOME" | sort -h; }
 export KDF_ITERS=100  # Much faster encryption
 
 # Prepare test
+# Isolate the smoke test from any WORK_TREE the user might have exported
+export WORK_TREE=
```

Comment:
```suggestion
unset WORK_TREE
```
There's potentially a minor difference between a variable being unset and being set to null. Makes it just a tad bit clearer that we don't use the variable as opposed to using it blank.